src/components/common/AmsTimeline.js

```diff
@@ -0,0 +1,169 @@
+import { Chip, Paper, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import SearchIcon from '@material-ui/icons/Search';
+import SettingsIcon from '@material-ui/icons/Settings';
+import {
+  Timeline,
+  TimelineConnector,
+  TimelineContent,
+  TimelineDot,
+  TimelineItem,
+  TimelineOppositeContent,
+  TimelineSeparator,
+} from '@material-ui/lab';
+import React from 'react';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    padding: theme.spacing(1),
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
+    width: '160px',
+  },
+  secondaryTail: {
+    backgroundColor: theme.palette.secondary.main,
+  },
+  oppositeContent: {
+    alignSelf: 'center',
+    padding: theme.spacing(1),
+  },
+  timelineContent: {
+    alignSelf: 'center',
+    padding: theme.spacing(1),
+  },
+  timelineItem: {
+    paddingLeft: theme.spacing(1),
+    paddingRight: theme.spacing(1),
+  },
+  timelineRoot: {
+    padding: theme.spacing(0),
+  },
+}));
+
+/**
+ * Renders a timeline component that includes all the processes and
+ * inspections associated with a crop cycle.
+ *
+ * @param {Array} processes All the crop cycles processes
+ * @param {Array} inspections All the inspections recorded on the crop cycle
+ */
+const AmsTimeline = ({ processes, inspections }) => {
+  const classes = useStyles();
+
+  const mergeProcessesAndInspections = () => {
```

Comment:
comment stating what the function is used for?

Reply:
Pretty trivial function and the name should give good enough indication of what is happening.